apps/core/src/router/index.tsx

```diff
@@ -0,0 +1,15 @@
+import { Layout } from "@zuri/ui";
+import { createBrowserRouter } from "react-router-dom";
+import { ErrorPage } from "../pages";
+import { getExternalRoutes } from "./routes";
+
+const router = createBrowserRouter([
+  {
+    path: "/",
+    element: <Layout hideSideBar />,
```

Comment:
the `hideSidebar` can be removed now since the sidebar visibility is managed by the state